Drop-In/src/main/res/values/styles.xml

```diff
@@ -32,6 +32,19 @@
         <item name="textColorError">@color/bt_error_red</item>
     </style>
 
+    <style name="bt_vault_manager_activity_theme" parent="Theme.AppCompat.NoActionBar">
+        <item name="windowNoTitle">true</item>
+        <item name="android:windowContentOverlay">@null</item>
+        <item name="android:windowBackground">@android:color/transparent</item>
+        <item name="android:colorBackgroundCacheHint">@null</item>
+        <item name="android:windowIsTranslucent">true</item>
+        <item name="android:windowTranslucentStatus" tools:targetApi="kitkat">true</item>
+        <item name="android:windowAnimationStyle">@style/bt_fade_activity_animation</item>
```

Comment:
was the slide animation too buggy? in the first gif there was some pretty bad clipping, but i think the fade is kinda jarring, unless we can get the bottom sheet to be the same height as the select payment method view